### PR TITLE
SDK-1210. Fixes for auto sync tests after sync management merge

### PR DESCRIPTION
**Pull request by mattw-mega**

### CODE REVIEW

File: tests/integration/SdkTest_test.cpp [hunk 1]
```diff
@@ -5035,10 +5053,10 @@ TEST_F(SdkTest, SyncResumptionAfterFetchNodes)
 
     LOG_verbose << " SyncResumptionAfterFetchNodes : syncying folders";
 
-    int tag1 = syncFolder(sync1Path);
+    int tag1 = syncFolder(sync1Path);  tag1;
```
Comment (catalinr-m) [inline tests/integration/SdkTest_test.cpp:5056]: Out of curiosity, what is this for?

Comment (polmr) [inline tests/integration/SdkTest_test.cpp:5056]: :upside_down_face: :smile:

File: tests/integration/SdkTest_test.cpp [hunk 2]
```diff
@@ -5107,7 +5125,7 @@ TEST_F(SdkTest, SyncResumptionAfterFetchNodes)
     removeSync(sync4Path);
 
     // wait for the sync removals to actually take place
-    std::this_thread::sleep_for(std::chrono::seconds{20});
+    std::this_thread::sleep_for(std::chrono::seconds{5});
```
Comment (polmr) [inline tests/integration/SdkTest_test.cpp:5128]: Note the time reduction before merging. Is it intentionally commited?

**Review by polmr — APPROVED** [review]

File: tests/integration/SdkTest_test.cpp (hunk 1 above)
Comment (polmr) [reply tests/integration/SdkTest_test.cpp:5056]: to avoid unused var warnings I guess :-D

Comment (catalinr-m) [reply tests/integration/SdkTest_test.cpp:5056]: IIRC that won't work when optimizations are enabled, i.e. Release builds will still get the warnings.
Using `(void)tag1;` should work for all cases.

**Review by polmr — DISMISSED** [review]

File: tests/integration/SdkTest_test.cpp (hunk 1 above)
Comment (polmr) [reply tests/integration/SdkTest_test.cpp:5056]: wouldn't it? Interesting ... I would have though those warnings were reported before the optimizations take place :thinking:

Comment (catalinr-m) [reply tests/integration/SdkTest_test.cpp:5056]: Probably (only?) the NOPs are optimized earlier.
At the end of the optimized build unused variables won't be present at all in the final executable, but the warning is there even for those cases, so not all optimization have been applied at that point.

**Review by sergiohs84 — APPROVED** [review]

File: tests/integration/SdkTest_test.cpp (hunk 2 above)
Comment (mattw-mega) [reply tests/integration/SdkTest_test.cpp:5128]: yes - the tests are pretty slow already, and 5 seconds should be more than enough :-)  thanks

**Pull request merged**